bedrock/settings/base.py

```diff
@@ -689,6 +689,20 @@ def facebook_tab_url_lazy():
 # Optimize.ly project code
 OPTIMIZELY_PROJECT_ID = None
 
+# Fx Accounts iframe source
+# Prod, stage, dev, & demos have this set in their local.py files.
+#
+# For local testing:
+#   - Point FXA_IFRAME_SRC to stomlinson's instance in your own local.py:
+#     FXA_IFRAME_SRC = 'https://stomlinson.dev.lcip.org/'
```

Comment:
A good question. If we end up keeping FxA in the firstrun experience, we'll definitely need a better way to test locally. There is an option to run the entire FxA stack locally, but it's arguably harder to get running that bedrock (and definitely more volatile), which makes it rather prohibitive for us bedrock folk.